packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -986,4 +998,8 @@ class ImportedBucket extends BucketBase {
   private generateDomainName() {
     return `${this.bucketName}.s3.amazonaws.com`;
   }
+
+  private generateBucketWebsiteUrl() {
+    return `${this.bucketName}.s3-website-${new cdk.Aws().region}.amazonaws.com`;
```

Comment:
Region change seems like a simple change 😄 

The URL suffix seems less so. Do you know which one it is depending on which region? It's a bit ugly but we could have something like:
```javascript
const suffix = cdk.Stack.find(this).urlSuffix
const region = cdk.Stack.find(this).region
const regionToSuffix = {
  'eu-west-1': `s3-website-eu-west-1.${suffix}`,
  'us-west-1': `s3-website.us-west-1.${suffix}`
}

return `${this.bucketName}.${regionToSuffix[region]}`;
```

Reply:
In fact, this problem is intractable for you right now (since we may be deploying without the region being available at all), so don't worry about it. This ties into https://github.com/awslabs/aws-cdk/issues/1282, we need some kind of region information to solve this.

Leave it as is and ignore it, I was just making a mental note of this issue.